### ISSUE TITLE
Investigate missing videos on IIS.NET

### ISSUE THREAD

**Issue by Rick-Anderson** [dotnet/AspNetCore.Docs#32554]
what happed to the media on learn.microsoft.com?
[https://www.iis.net/](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.iis.net%2F&data=05%7C02%7CRick.Anderson%40microsoft.com%7C9f8d6d033ff6493fb15008dc7060399b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638508805793200070%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=SI3FPqP7A7qE5IeRMKgrprisBgTEglCNHz7moq%2FYJ9M%3D&reserved=0) links to several videos that are now gone. For example: [https://learn.microsoft.com/video/media/373b405a-7259-4786-951e-4d4511bd0fd1/webdeploy_high.mp4](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fvideo%2Fmedia%2F373b405a-7259-4786-951e-4d4511bd0fd1%2Fwebdeploy_high.mp4&data=05%7C02%7CRick.Anderson%40microsoft.com%7C9f8d6d033ff6493fb15008dc7060399b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638508805793209778%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=hcYLo3Njb4hHHVfJIS9k92OeqignTtN56y%2BpHXaUj%2BU%3D&reserved=0).

Any idea what happened to the videos?

Check on:

- [x] Teams 
- [x] Create a work item  (Note: iis-docs does not allow tracking work items/issues to be made, so unless the vendor that was suggested to fix the issues is using DevOps, they will just have to refer to this issue.  -WadePickett)

---
[Associated WorkItem - 255194](https://dev.azure.com/msft-skilling/Content/_workitems/edit/255194)

**Comment by wadepickett** [comment]
@Rick-Anderson, for the example video you provided. Is that an example of the area of videos, but that video is working fine, or was that an example of a problem video.  I am able to reach that video and play it fine, so I assume the former, but I'll ask just to verify we are on the same page, thanks:

[https://learn.microsoft.com/video/media/373b405a-7259-4786-951e-4d4511bd0fd1/webdeploy_high.mp4](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fvideo%2Fmedia%2F373b405a-7259-4786-951e-4d4511bd0fd1%2Fwebdeploy_high.mp4&data=05%7C02%7CRick.Anderson%40microsoft.com%7C9f8d6d033ff6493fb15008dc7060399b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638508805793209778%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=hcYLo3Njb4hHHVfJIS9k92OeqignTtN56y%2BpHXaUj%2BU%3D&reserved=0).

**Comment by Rick-Anderson** [comment]
> @Rick-Anderson, for the example video you provided. Is that an example of the area of videos, but that video is working fine, or was that an example of a problem video. I am able to reach that video and play it fine, so I assume the former, but I'll ask just to verify we are on the same page, thanks:
> 
> [https://learn.microsoft.com/video/media/373b405a-7259-4786-951e-4d4511bd0fd1/webdeploy_high.mp4](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fvideo%2Fmedia%2F373b405a-7259-4786-951e-4d4511bd0fd1%2Fwebdeploy_high.mp4&data=05%7C02%7CRick.Anderson%40microsoft.com%7C9f8d6d033ff6493fb15008dc7060399b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638508805793209778%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=hcYLo3Njb4hHHVfJIS9k92OeqignTtN56y%2BpHXaUj%2BU%3D&reserved=0).

Obviously the video is fine, check with  @John-Hart

**Comment by wadepickett** [comment]
Yes, well, obviously.  :)  It was not clear why you sent that one, but I found the thread and more context, thanks.

**Comment by wadepickett** [comment]
I'll add some notes here as I go:

Looks like some videos were migrated to here: https://review.learn.microsoft.com/en-us/shows/iis-net-site-videos/?branch=main&terms=routing

For example, this one that is still linked at CH9:
https://sec.ch9.ms/ch9/fbcf/27db9b38-0e8d-4dec-99c1-ddda370afbcf/applicationrequestrouting_high.mp4
is also on learn,microsoft.com:
https://review.learn.microsoft.com/en-us/shows/iis-net-site-videos/application-request-routing?branch=main

Still checking into it of course...

**Comment by wadepickett** [comment]
I found the ​commit where it looks like the broken CH9 links were removed in mass.  42 video content pages had their  channel9.msdn.com link removed in 2021 here:
[Old channle9 video link removed · MicrosoftDocs/iis-docs@a760974 (github.com)](https://github.com/MicrosoftDocs/iis-docs/commit/a760974829f669785ac49785a6d6b9610e294311)


For example:
on:
iis/application-frameworks/running-php-applications-on-iis/mix08-partying-with-php-on-microsoft-internet-information-services-7-and-above.md

This link was removed:
https://channel9.msdn.com/Blogs/IIS-NET-Site-Videos/MIX08-Partying-with-PHP-on-Microsoft-Internet-Information-Services-7

But instead, it should have been updated to where it migrated to Learn at:
https://learn.microsoft.com/en-us/shows/iis-net-site-videos/mix08-partying-php-on-microsoft-internet-information-services-7.

Will create a new issue once some questions are answered on the other thread for this.  It is iis-docs specific and essentially a reversal of the commit mentioned above with the links updated that were deleted to the new learn location from what I see so far.
